docs/development/integration-tests.md

```diff
@@ -1,14 +1,25 @@
 # Integration Tests
-Cumulus has a comprehensive set of integration tests that tests the framework on an active AWS account. As long as you have an AWS account with valid credentials you can run the tests as described below:
+Cumulus has a comprehensive set of integration tests that tests the framework on
+an active AWS account. As long as you have an AWS account with valid credentials
+you can run the tests as described below:
 
 ## Running integration tests on AWS
 
-- Run `./bin/prepare`
-- Deploy your instance integrations on aws and run tests by following the steps [here](https://github.com/nasa/cumulus/tree/master/example/README.md)
+- Run:
```

Comment:
How is `./bin/prepare` no longer necessary?

Reply:
The `example` package was added to the lerna config [here](https://github.com/nasa/cumulus/blob/34695996189bf9670cc49d5aed1e1e9d56a9fdf3/lerna.json#L5).  `npm run bootstrap` now installs (and links) the dependencies for the `example` package.  It also builds all of the packages, which was the other thing that `./bin/prepare` was doing.

As a side note, I had to add `example` to lerna's config because npm handles package linking differently than yarn (and lerna) do.  lerna and yarn created links from a package's `node_modules` directory to the directory of the package that it depended on.  `npm link` installed the package to the global package directory, then linked to that.  I forget exactly what the failure was, but the difference in linking was breaking the build.

Reply:
Just to be clear, was it necessary to move away from `./bin/prepare` to do package linking into the `example` directory with npm? Or was this a simplification?

If it was the latter, as much as I like deleting code, I'd rather keep `./bin/prepare` around. I feel like treating `example` as a package might be opening us up to more problems. Perhaps it is related to this other issue you mentioned below?

https://github.com/nasa/cumulus/pull/871/files#r265274338

Reply:
`./bin/prepare` was no longer necessary because of package linking in `./example` now being handled by lerna.